### ISSUE TITLE
git-remote-hg doesn't fetch

### ISSUE THREAD

**Issue by electronicsRules** [felipec/git#70]
Git version 2.1.3
Mercurial version 3.2.1
Python version 2.7.7
SHA1: 096f517bd78982d82e333e0779dfc6fd0b2e0929  git-remote-hg
Sabayon Linux 64-bit, kernel version 3.15.10
```
$ git clone "hg::http://selenic.com/repo/hello"
Cloning into 'hello'...
Traceback (most recent call last):
  File "/home/georgiy/bin/git-remote-hg", line 1322, in <module>
    sys.exit(main(sys.argv))
  File "/home/georgiy/bin/git-remote-hg", line 1286, in main
    repo = get_repo(url, alias)
  File "/home/georgiy/bin/git-remote-hg", line 433, in get_repo
    repo.pull(peer, heads=None, force=True)
  File "/usr/lib64/python2.7/site-packages/mercurial/repoview.py", line 307, in __getattr__
    return getattr(self._unfilteredrepo, attr)
AttributeError: 'localrepository' object has no attribute 'pull'
```

**Comment by fingolfin** [comment]
This is an incompatibility with Mercurial >= 3.2. This has been reported before here:
  https://github.com/felipec/git-remote-hg/issues/27
For a fix, see also here:
  https://github.com/felipec/git-remote-hg/pull/28
or just use my branch of git-remote-hg which fixes the issue, too:
  https://github.com/fingolfin/git-remote-hg

But now that Felipe is back, I guess he'll soon have a fix for this in his repo, too.

**Comment by felipec** [comment]
Should be fixed in v2.0.0+fc4.
